src/Frame.js

```diff
@@ -0,0 +1,41 @@
+var Frame = function() {
+  this.frameScore = 0;
+  this.skittlesRemaining = 10;
+};
+
+Frame.prototype.play = function() {
+  score = this.bowl();
+  this.frameScore += score;
+  this.skittlesRemaining -= score;
+  if(this.skittlesRemaining === 0) {
+    return 'Strike';
+  }
+  score = this.bowl();
+  this.frameScore += score;
+  this.skittlesRemaining -= score;
+  if(this.skittlesRemaining === 0) {
+    return 'Spare';
+  } else {
+    return '' + this.frameScore;
```

Comment:
If you are going to set state with this somehow, can I suggest that you set it to one of three strings: `'Strike'`, `'Spare'`, or `'Open'` - then track the previous score in a different section of state